### PR TITLE
feat: add loading spinner while resolving ens name

### PR DESCRIPTION
**Pull request by hydr063n**
fixes https://github.com/status-im/nim-status-client/issues/1084

Peek 2020-11-13 17-16

### CODE REVIEW

**Comment by PascalPrecht** [comment]
Nice PR!

File: ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml [hunk 1]
```diff
@@ -166,26 +171,24 @@ ModalPopup {
         }
     }
 
-    footer: Button {
-        width: 44
-        height: 44
+    footer: Item {
```
Comment (richard-ramos) [inline ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml:174]: I wonder if we could just use StatusButton directly here

**Review by richard-ramos — APPROVED** [review]
good work!

File: ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml (hunk 1 above)
Comment (jrainville) [reply ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml:174]: Yes we can. Also, only the `anchor.right` is necessary

**Review by jrainville — COMMENTED** [review]
Looks good. I'd just remove the `Item` in the footer like Richard proposes just so it's a bit cleaner.

**Review by jrainville — APPROVED** [review]
Nice work

File: ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml (hunk 1 above)
Comment (hydr063n) [reply ui/app/AppLayouts/Chat/components/PrivateChatPopup.qml]: Yes, thats a much cleaner approach. I've added both this changes!

**Pull request merged**